Shared/Samples/Apply class breaks renderer to sublayer/ApplyClassBreaksRendererToSublayerView.swift

```diff
@@ -0,0 +1,123 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyClassBreaksRendererToSublayerView: View {
+    /// The map image layer.
+    private let mapImageLayer = ArcGISMapImageLayer(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer")!)
```

Comment:
I'm fairly certain this needs to be a state variable or the layer will get re-created anytime the view redraws. If you background the app, hit a button, there are ways the view can redraw and when it does it will re-create this because it's not state.